tailfs/automount_darwin.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build darwin
+
+package tailfs
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+	"strconv"
+)
+
+// DefaultAutomountPath returns the default automount path. If blank, that
+// means TailFS is disabled on this platform.
+func DefaultAutomountPath() string {
+	return "/Volumes/tailscale"
+}
+
+func MountShares(location string, username string) {
+	u, err := user.Lookup(username)
+	if err != nil {
+		log.Printf("warning: error looking up user %q, won't automount shares: %s", username, err)
+		return
+	}
+	uid, err := strconv.Atoi(u.Uid)
+	if err != nil {
+		log.Printf("warning: failed to parse uid %q, won't automount shares: %s", u.Uid, err)
+	}
+	gid, err := strconv.Atoi(u.Gid)
+	if err != nil {
+		log.Printf("warning: failed to parse gid %q, won't automount shares: %s", u.Gid, err)
+	}
```

Comment:
Shouldn't we be returning on error here given that we are passing uid and gid into os.Chown? If there are no returning `os.Chown(location, 0, 0)` will cause unintended effects of assigning ownership to the root user. At least please document this condition, even though it is very unlikely to occur.